src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,495 @@
+    #     in_source_only_direct_source_edges, source_data_map)
+
+    # - indirect <--> indirect
+    #   - Indirect SCRs that exist in both Mondo and source
+    # in_both_indirect_mondo_edges: Set[RELATIONSHIP] = (
+    #     rels_indirect_source_mondo.intersection(rels_indirect_mondo_mondo))
+    # in_both_indirect: List[Dict] = _edges_with_metadata_from_plain_edges(in_both_indirect_mondo_edges, mondo_data_map)
+    #   - Indirect SCRs that exist in source but not in Mondo
+    # in_source_only_indirect_mondo_edges: Set[RELATIONSHIP] = rels_indirect_source_mondo.difference(
+    #     rels_indirect_mondo_mondo)
+    # in_source_only_indirect: List[Dict] = _edges_with_metadata_from_plain_edges(
+    #     in_source_only_indirect_mondo_edges, mondo_data_map)
+    #   - Indirect SCRs that exist in Mondo but not in source
+    # in_mondo_only_indirect_mondo_edges: Set[RELATIONSHIP] = rels_indirect_mondo_mondo.difference(
+    #     rels_indirect_source_mondo)
+    # in_mondo_only_indirect: List[Dict] = _edges_with_metadata_from_plain_edges(
+    #     in_mondo_only_indirect_mondo_edges, mondo_data_map)
+
+    # - direct <--> ancestor
+    #   - Direct SCRs in source that don't exist at all in Mondo
+    in_source_direct_not_in_mondo_source_edges = rels_direct_source_source.difference(ancestors_mondo_source)
+    in_source_direct_not_in_mondo: List[Dict] = _edges_with_metadata_from_plain_edges(
+        in_source_direct_not_in_mondo_source_edges, source_data_map)
+    #   - Direct SCRs in Mondo that don't exist at all in source
+    in_mondo_direct_not_in_source_mondo_edges: Set[RELATIONSHIP] = (
+        rels_direct_mondo_mondo.difference(ancestors_source_mondo))
+    # in_mondo_direct_not_in_source: List[Dict] = _edges_with_metadata_from_plain_edges(
+    #     in_mondo_direct_not_in_source_mondo_edges, mondo_data_map)
+
+    # Create outputs ---------------------------------------------------------------------------------------------------
+    # Google doc about cases:
+    # https://docs.google.com/document/d/1H8fJKiKD-L1tfS-2LJu8t0_2YXJ1PQJ_7Zkoj7Vf7xA/edit#heading=h.9hixairfgxa1
+    logging.info('Creating outputs for synchronization cases')
+    common_sort_cols = ['subject_mondo_id', 'object_mondo_id', 'subject_source_id', 'object_source_id']
+
+    # Case 1: SCR direct in source and Mondo
+    subheader = deepcopy(ROBOT_SUBHEADER)
+    subheader[0]['object_mondo_id'] = 'SC %'
+    df1 = pd.DataFrame(in_both_direct)
+    if len(df1) == 0:
+        df1 = pd.DataFrame(columns=list(subheader[0].keys()))
+    df1 = df1.sort_values(common_sort_cols)
+    df1 = pd.concat([pd.DataFrame(subheader), df1])
+    df1.to_csv(outpath_confirmed, sep='\t', index=False)
+
+    # Case 2: SCR is direct in source, but indirect Mondo
+    pass  # no output for this case
+
+    # Case 3: SCR is direct in the source, but not at all in Mondo
+    subheader = deepcopy(ROBOT_SUBHEADER)
+    subheader[0]['object_mondo_id'] = 'AI MONDO:excluded_subClassOf'
```

Comment:
## Dual bugfix on case 3 'added' template
Addresses:
- [x] `object_mondo_id` is currently `SC %` in non-obsolete template
- [x]  `AI ` missing from `MONDO:excluded_subClassOf`

I did the bug fixes but I did not have the time to re-run the outputs with my limited time this weekend and with the urgent work on fixing the pipeline.

Reply:
```suggestion
    subheader[0]['object_mondo_id'] = 'AI obo:mondo#excluded_subClassOf'
```

This was not the exactly correct IRI, but this one is (I also [tested it](https://github.com/monarch-initiative/mondo/pull/7111)).

Reply:
Oh, very good! Thanks I'll change that now.

Reply:
Trish and I decided today it was time to merge this!